securedrop_client/gui/widgets.py

```diff
@@ -2244,6 +2244,9 @@ def __init__(
         file_ready_signal.connect(self._on_file_downloaded, type=Qt.QueuedConnection)
         file_missing.connect(self._on_file_missing, type=Qt.QueuedConnection)
 
+    def update_file_size(self):
+        self.file_size.setText(humanize_filesize(self.file.size))
```

Comment:
Note: this could cause a crash if the file was deleted, I believe, so we can either add it to [our ticket tracking possible crashers](https://github.com/freedomofpress/securedrop-client/issues/906#issuecomment-601410639) or temporary add a try catch for now.

Reply:
actually we should do both - add the try-catch and add it to the list

Reply:
I tried adding a test that created the `FileWidget`, then deleted its `File`, then called `update_file_size` and it did not crash. I've added the try/except block and a test with a mock to force an exception, just in case.